design.md

```diff
@@ -288,12 +288,20 @@ inspiration from.
    schemas, need to be explicitly turned on in configuration, and follow
    all the other expectations we have around eventlogging data.
 
-5. *User consent / information UI*
+5. *User consent / information API and example UI*
 
    Every application collecting data should have a way to make it
    clear to the user what is being collected, and possibly ways
    to turn it off. We could possibly let admins configure opt-in /
-   opt-out options.
+   opt-out options. In some cases, we may wish to store consent information
+   separately from event data (this is commondly done in human subjects research),
+   so the API should use a separately configured store for this information.
+   
+   By providing an API with a reference UI, a consent process can be
+   integrated into a variety of views into Jupyter ecosystem products
+   (e.g., JupyterHub, JupyterLab, Binder) as well as "independent"
+   projects like Nteract, Stencila, or Gigantum, or even completely
+   distinct projects like Theia or RStudio.
 
```

Comment:
Suggesting a minor changing of phrasing. If this changes the interpretation, feel free to ignore. 

> The "consent and information" API should provide a reference UI that can be integrated with various Jupyter-based applications (e.g. JupyterHub, JupyterLab, Binder, Nteract, Stencila, Gigantum, etc.) and non-Jupyter applications (e.g. RStudio and Theia).

Reply:
Took your suggestions with a minor "Jupyter-based" -> "Jupyter-related". I like the reduction in complexity.